examples/remote-ui/README.md

```diff
@@ -0,0 +1,11 @@
+# Remote UI example
```

Comment:
Did you consider just adding the remote UI example alongside the DOM-based remote environments in the other example? I am not sure if it would be nicer across the board, but it would be nice to avoid duplicating most of the host-side code between the two (and it helps reinforce that you can use Remote DOM exclusively on the host, while having remote-ui and Remote DOM running in the sandbox).

Reply:
I briefly looked into it and seemed kinda complicated for reasons I don't recall anymore tho. 😅 

I figured it's easier to just have a second example. I also assumed that we'd probably remove the remote-ui adapter and example at some point in the not too distant future.